entities/Rules/index.js

```diff
@@ -133,34 +133,34 @@ module.exports = {
   ],
   PAY_FINE: [
     ({ UI }) => UI.payFine(),
-    // TODO: WealthService
     (_, gameState) => {
-      gameState.currentPlayer.cash -= gameState.config.fineAmount;
+      require('../WealthService').decrement(
+        gameState.currentPlayer,
+        gameState.config.fineAmount
+      );
       gameState.currentPlayer.jailed = -1;
-      // TODO: investigate using setter for bankruptcy logic
     },
     function conditionalEventsOnLostWealth({ notify }, gameState) {
```

Comment:
Do we need a rule-check for "[if a player can not pay another player the banks pays]" ? Because we are doing this check after decrementing, it would mean that if the "renter" couldn't afford their rent (even after liquidating and all), they had "created" money for the "landlord" as now

Reply:
We will in the future case, but for this one, it's a fine to the bank. If they don't have enough money, they liquidate or lose. Adding a point to review that in the rent function in the next PR.